feature/home/src/main/java/com/mashup/dorabangs/feature/home/HomeViewModel.kt

```diff
@@ -48,21 +46,32 @@ class HomeViewModel @Inject constructor(
     val savedStateHandle: SavedStateHandle,
     private val getLastCopiedUrlUseCase: GetLastCopiedUrlUseCase,
     private val getFolderList: GetFolderListUseCase,
-    private val getPostsUseCase: GetPosts,
+    private val getPostsPageUseCase: GetPostsPageUseCase,
     private val setLastCopiedUrlUseCase: SetLastCopiedUrlUseCase,
     private val createFolderUseCase: CreateFolderUseCase,
     private val saveLinkUseCase: SaveLinkUseCase,
-    private val getSavedLinksFromFolderUseCase: GetSavedLinksFromFolderUseCase,
+    private val getPostsFromFolderUseCase: GetPostsFromFolderUseCase,
     private val getUnReadPostsCountUseCase: GetUnReadPostsCountUseCase,
     private val getAIClassificationCount: GetAIClassificationCountUseCase,
     private val getIdFromLinkToReadLaterUseCase: GetIdFromLinkToReadLaterUseCase,
     private val setIdFromLinkToReadLaterUseCase: SetIdLinkToReadLaterUseCase,
     private val deletePostUseCase: DeletePostUseCase,
     private val changePostFolderUseCase: ChangePostFolder,
     private val patchPostInfoUseCase: PatchPostInfoUseCase,
+    private val getPostUseCase: GetPostUseCase,
 ) : ViewModel(), ContainerHost<HomeState, HomeSideEffect> {
     override val container = container<HomeState, HomeSideEffect>(HomeState())
 
+    val postList = mutableStateListOf<FeedUiModel.FeedCardUiModel>()
+
```

Comment:
private이랑 외부에서는 가져다 사용할 수 있게 두개 구분해두면 되지 않을까~? 이게 고민하고 있는게 맞낭
```suggestion
    val _postList = mutableStateListOf<FeedUiModel.FeedCardUiModel>()
   val postList = _postList.asStateFlow() 

```

Reply:
@ddyeon 
mutalbeStateListOf가 Flow가 아니라 stateFlow로 바꾸지는 못함 ㅠ